index.bs

```diff
@@ -2367,6 +2449,15 @@ To <dfn noexport>parse source-registration JSON</dfn> given a [=byte sequence=]
 1. Let |triggerSpecs| be the result of [=parsing trigger specs=] with |value|,
     |sourceTime|, |sourceType|, |expiry|, and |triggerDataMatchingMode|.
 1. If |triggerSpecs| is an error, return null.
+1. Let |attributionScopeLimit| be null.
```

Comment:
As attribution_scope_limit is required when either max_event_states or attribution_scopes is set. It seems that it may be easier to wrap them in a dictionary field.
{
  …,
   “attribution_scope”: {
      “limit”: 5, // required field
      “scopes”: [“a”], // optional field
      “max_event_states”: 123 // optional field 
   }
}

Reply:
I don't think adding an extra layer would give us much benefit tbh, other than more complexity for user experience and implementation.

(and it might not be a good time for us change external facing field structure at this stage, not the main reason though)